crates/sui-core/src/module_cache_metrics.rs

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Mysten Labs, Inc.
```

Comment:
I guess git did not catch my rename of the file above. Not a big deal though. Bottom line the implementation of the gauge above is replaced by this file